### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by avigoldman**
Migrating the changelog to a new format.

- [ ] Only generate individual pages for new changes
- [ ] Add a visual difference (anchor icon) when there is a page for a single change

### CODE REVIEW

File: content/pages/changelog.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ *
+ * This is the changelog page. It shows all of the changes with the all details
+ * collapsed.
+ *
+ * Each individual change is generated into it's own page using the template
+ * located at src/template/changelog.js
+ */
+
+import React from 'react'
+import Helmet from 'react-helmet'
```
Comment (jgzamora) [inline content/pages/changelog.js]: 👷

File: content/pages/changelog.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      />
+      <Section light>
+        <Container>
+          <Row center="xs">
+            <Column md={7} sm={10} xs={11}>
+              <TopSection>
+                <h1>Changelog</h1>
+                <p>
+                  A running log of what's new and what's been changed in
+                  SparkPost.
```
Comment (jgzamora) [inline content/pages/changelog.js]: Is the new line here just prettier? The other one doesn't have it

File: src/components/Anchor/index.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import React from 'react'
 import styled from 'styled-components'
 import slugify from 'slugify'
+import { castArray } from 'lodash'
 import Link from 'components/Link'
 
 function Anchor({ title, children, ...props }) {
+  if (!title) title = castArray(children).join('')
```
Comment (jgzamora) [inline src/components/Anchor/index.js]: Too modern 😠

File: src/components/Change/index.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+const markdownComponents = {
+  h1({ children, ...props }) {
+    return <h4 {...props}>{children}</h4>
+  },
+  h2({ children, ...props }) {
+    return <h5 {...props}>{children}</h5>
+  },
+  h3({ children, ...props }) {
+    return <h5 {...props}>{children}</h5>
```
Comment (jgzamora) [inline src/components/Change/index.js]: pls explain `h2` & `h3` = `h5` in chat

File: src/components/Change/index.js [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+  i {
+    transition: transform .15s;
+
+    ${props => props.isActive && `
+      transform: rotate(90deg);
+    `}
+  }
+`
+
+class Change extends Component {
```
Comment (jgzamora) [inline src/components/Change/index.js]: this class (or maybe the file) could use a comment

File: src/templates/changelog.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
+import React from 'react'
```
Comment (jgzamora) [inline src/templates/changelog.js]: Similar comment to the other changelog file pls

File: src/components/Change/index.js (hunk 4 above)
Comment (avigoldman) [reply src/components/Change/index.js]: We don't want any titles larger than `h4` in the change contents.

So if we have any, we shrink them down to `h4` or `h5`. But we also want them to change relative to their original size.

So we shift `h1` to `h4` and `h2`/`h3` to `h5`.

**Comment by nicklemmon** [comment]
Closing stale PR.

**Pull request closed without merging**